tests/turbine_multi_dim_unit_test.py

```diff
@@ -169,7 +162,7 @@ def test_power():
         multidim_condition=condition
     )
 
-    power_truth = 3029825.10569982
+    power_truth = 12424759.67683091
```

Comment:
Is this 4x increase expected and what the cause of it?

Reply:
Yes---this is because there was a weird mix of the NREL 5MW turbine model with the IEA 15MW floating Cp Ct tables before; now it is consitently the IEA 15MW that is used in this test.